### PR TITLE
[CH] Support conv function

### PR DESCRIPTION
**Pull request by zhanglistar**
## What changes were proposed in this pull request?

(Please fill in changes proposed in this fix)

(Fixes: \#2348)

## How was this patch tested?

(Please explain how this patch was tested. E.g. unit tests, integration tests, manual tests)
UT

(If this patch involves UI changes, please attach a screenshot; otherwise, remove this)

### CODE REVIEW

**Review by zzcclp — APPROVED** [review]
LGTM

File: cpp-ch/local-engine/Functions/SparkFunctionConv.cpp [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+ */
+#include "SparkFunctionConv.h"
+#include <string>
+#include <DataTypes/DataTypeNullable.h>
+#include <Functions/FunctionFactory.h>
+#include <Poco/Logger.h>
+#include <Poco/Types.h>
+#include <Common/Exception.h>
+#include <Common/logger_useful.h>
+#include "DataTypes/IDataType.h"
```
Comment (liuneng1994) [inline cpp-ch/local-engine/Functions/SparkFunctionConv.cpp:25]: use <>

File: cpp-ch/local-engine/Functions/SparkFunctionConv.cpp [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+        throw DB::Exception(DB::ErrorCodes::NUMBER_OF_ARGUMENTS_DOESNT_MATCH,
+            "Number of arguments for function {} doesn't match: passed {}, should be 3.",
+            getName(), arguments.size());
+
+    if (!isInteger(arguments[1]))
+        throw DB::Exception(DB::ErrorCodes::ILLEGAL_TYPE_OF_ARGUMENT,
+            "Second argument for function {} must be Int", getName());
+    if (!isInteger(arguments[2]))
+        throw DB::Exception(DB::ErrorCodes::ILLEGAL_TYPE_OF_ARGUMENT,
+            "Thrid argument for function {} must be Int", getName());
```
Comment (liuneng1994) [inline cpp-ch/local-engine/Functions/SparkFunctionConv.cpp:52]: typo third

**Review by liuneng1994 — APPROVED** [review]
LGTM

**Review by zzcclp — APPROVED** [review]
LGTM

**Pull request merged**